src/Chess.cpp

```diff
@@ -0,0 +1,257 @@
+	#include "Chess.h"
+
+// clear the screen "cls"
+void Chess::clear() const 
+{
+	COORD topLeft = { 0, 0 };
+	HANDLE console = GetStdHandle(STD_OUTPUT_HANDLE);
+	CONSOLE_SCREEN_BUFFER_INFO screen;
+	DWORD written;
+
+	GetConsoleScreenBufferInfo(console, &screen);
+	FillConsoleOutputCharacterA(
+		console, ' ', screen.dwSize.X * screen.dwSize.Y, topLeft, &written
+	);
+	FillConsoleOutputAttribute(
+		console, FOREGROUND_GREEN | FOREGROUND_RED | FOREGROUND_BLUE,
+		screen.dwSize.X * screen.dwSize.Y, topLeft, &written
+	);
+	SetConsoleCursorPosition(console, topLeft);
+}
+// create the GUI - ASCII art
+void Chess::setFrames() 
+{ 
+	// set all to ' ' (space bar ascii value 32) instead of 0  
+	{
+		for (size_t row = 0; row < _SIZE; ++row)
+			for (size_t col = 0; col < _SIZE; ++col)
+				m_board[row][col] = 32;
+	}
+	// set out frame
+	{
+		m_board[0][0] = 201;  m_board[0][20] = 187;
+		m_board[20][0] = 200; m_board[20][20] = 188;
+
+		for (size_t i = 1; i < 20; ++i)
+		{
+			m_board[0][i] = 205;
+			m_board[20][i] = 205;
+			m_board[i][0] = 186;
+			m_board[i][20] = 186;
+		}
+	} 
+
+	// set in side frame 
+	{
+		m_board[2][2] = 218;  m_board[2][18] = 191;
+		m_board[18][2] = 192; m_board[18][18] = 217;
+
+		for (size_t i = 4; i < 17; i += 2)
+		{
+			m_board[2][i] = 194;
+			m_board[18][i] = 193;
+			m_board[i][2] = 195;
+			m_board[i][18] = 180;
+		}
+
+		for (size_t i = 2; i < 19; i += 2)
+			for (size_t j = 3; j < 19; j += 2)
+				m_board[i][j] = 196;
+		
+		for (size_t i = 3; i < 18; i += 2)
+			for (size_t j = 2; j < 19; j += 2)
+				m_board[i][j] = 179;
+
+		for (size_t i = 4; i < 17; i += 2)
+			for (size_t j = 4; j < 17; j += 2)
+				m_board[i][j] = 197;
+
+		for (size_t i = 4; i < 17; i += 2)
+			m_board[2][i] = 194;
+		for (size_t i = 4; i < 17; i += 2)
+			m_board[18][i] = 193;
+		for (size_t i = 4; i < 17; i += 2)
+			m_board[i][2] = 195;
+		for (size_t i = 4; i < 17; i += 2)
+			m_board[i][18] = 180;
+	}
+
+	// set nums and letters 
+	{
+		// nums 
+		for (size_t i = 3, t = 0; i < 19; i += 2, ++t)
+			m_board[1][i] = m_board[19][i] = ('1' + t);
+		// letters 
+		for (size_t i = 3, t = 0; i < 19; i += 2, ++t)
+			m_board[i][1] = m_board[i][19] = ('A' + t);
+	}
```

Comment:
Use more descriptive variable names to enhance code readability. For example, instead of using single-letter variable names like t, use names like rowIndex or columnIndex to convey the purpose of the variable.